src/main/resources/data.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO product (name, price)
```

Comment:
uruchamianie data.sql - dzieje się domyślnie przed stworzeniem tabel, można uzyć import.sql zamiast: najlepiej poczytaj tutaj: https://docs.spring.io/spring-boot/how-to/data-initialization.html#howto.data-initialization.using-hibernate